### PR TITLE
Doc: Reposition worker-utilization in doc

### PR DESCRIPTION
**Pull request by karenzone**
Fixes: #16275 

**PREVIEW:** https://logstash_bk_16335.docs-preview.app.elstc.co/guide/en/logstash/master/node-stats-api.html

### CODE REVIEW

**Review by kaisecheng — COMMENTED** [review]
It is a good try. You moved worker-utilization to "Plugin flow rates". 
worker-utilization also appear in "Pipeline flow rates". The detail description mainly explain what the data means to a pipeline. So, I think it will be more clear to relocate the description to [Pipeline flow rates](https://www.elastic.co/guide/en/logstash/8.13/node-stats-api.html#pipeline-flow-rates).

**Comment by karenzone** [comment]
@kaise:  Makes sense.  Please see what you think.

File: docs/static/monitoring/monitoring-apis.asciidoc [hunk 1]
```diff
@@ -1002,6 +985,18 @@ A positive number indicates that the queue size-on-disk is growing, and a negati
 
 NOTE: The size of a PQ on disk includes both unacknowledged events and previously-acknowledged events from pages that contain one or more unprocessed events.
       This means it grows gradually as individual events are added, but shrinks in large chunks each time a whole page of processed events is reclaimed (read more: <<garbage-collection, PQ disk garbage collection>>).
+
+| worker_utilization | Filters, Outputs |
```
Comment (kaisecheng) [inline docs/static/monitoring/monitoring-apis.asciidoc:989]:
```suggestion
| worker_utilization |
```

**Review by kaisecheng — APPROVED** [review]
Left one suggestion, otherwise, LGTM

**Comment by karenzone** [comment]
@logstashmachine backport 8.15 8.14

**Comment by karenzone** [comment]
11472

**Comment by karenzone** [comment]
@logstashmachine backport 8.15

**Pull request merged**